lib/standard/string.nit

```diff
@@ -2207,6 +2216,12 @@ extern class NativeString `{ char* `}
 	# Creates a new NativeString with a capacity of `length`
 	new(length: Int) is intern
 
+	# Returns a char* starting at `index`.
+	#
+	# WARNING: Unsafe for extern code, use only for temporary
```

Comment:
Nope, that's just me being inconsistent for the NOTE/WARNING stuff, I'll change it.

As for the private, since `NativeString` is low-level stuff, the same reasoning as for `NativeArray` applies, if you choose to go though this class, you should always be aware of potential dangers.